libs/pages/settings/src/lib/feature/page-organization-billing-summary-feature/page-organization-billing-summary-feature.tsx

```diff
@@ -1,48 +1,45 @@
-import { ClusterStateEnum } from 'qovery-typescript-axios'
+import { OrganizationCurrentCost } from 'qovery-typescript-axios'
```

Comment:
```suggestion
import { type OrganizationCurrentCost } from 'qovery-typescript-axios'
```
For all type import we need to prefix with `type` to avoid linters errors, it's one of our typescript rule.

You can see why here, it's an optimization for our transpilers:
https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/consistent-type-imports.md